src/ResourceManager/Network/Commands.Network/help/Update-AzureRmP2SVpnServerConfiguration.md

```diff
@@ -0,0 +1,368 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/update-azurermp2svpnserverconfiguration
+schema: 2.0.0
+---
+
+# Update-AzureRmP2SVpnServerConfiguration
+
+## SYNOPSIS
+Updates an existing P2SVpnServerConfiguration associated with a VirtualWan.
+
+## SYNTAX
+
+### ByVirtualWanNameDefault (Default)
+```
+Update-AzureRmP2SVpnServerConfiguration [-VpnProtocol <String[]>] [-AsJob]
```

Comment:
This is still wrong, please regenerate

Reply:
Done.

Reply:
I have regenerated help files. Can you check if this is fixed? or Let me know what issue.

Reply:
This is still wrong - when you regenerated the file it should no longer have this "ByVirtualWanNameDefault" parameter set - that was only there because of the incorrect naming of the defaultparametersetname.  Please be sure to rebuild and then regenerate this file.

Reply:
This is still wrong - when you regenerated the file it should no longer have this "ByVirtualWanNameDefault" parameter set - that was only there because of the incorrect naming of the defaultparametersetname. Please be sure to rebuild and then regenerate this file.